chat-commands.js

```diff
@@ -3409,6 +3409,21 @@ exports.commands = {
 				group: targetUser.group,
 				rooms: roomList,
 			};
+			if (targetUser in room.users && user.can('addhtml', null, room)) {
```

Comment:
This version still allows global bots to check challenges by sending the command to the global room,
which is what bots most certainly always do.